src/doc/unstable-book/src/compiler-flags/self-profile-events.md

```diff
@@ -41,7 +41,7 @@ $ rustc -Zself-profile -Zself-profile-events=default,args
 
 - `llvm`
   - Adds tracing information about LLVM passes and codegeneration.
-  - Disabled by default because this only works when `-Znew-llvm-pass-manager` is enabled.
+  - Disabled by default for historical reasons.
```

Comment:
I know this is a cop-out -- should we just enable it?

Reply:
I do not think that is shall be enabled by default. it is also not enabled for the same reason as query-keys and function-args that the traces gets very large.
```suggestion
  - Disabled by default because this significantly increases the trace file size.
```

Reply:
Sounds good, thanks!